frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,34 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
-  return (
-    <CardTitle>
-      <span style={{ verticalAlign: 'text-bottom' }}>{odhApp.spec.displayName}</span>
-      {isRedHatSupported(odhApp) && (
+  let title = odhApp.spec.displayName;
+  let icon;
+
+  if (isRedHatSupported(odhApp)) {
+    const splitTitle = odhApp.spec.displayName.split(' ');
+    title = `${splitTitle.slice(0, -1).join(' ')} `;
+    icon = (
+      <span style={{ whiteSpace: 'nowrap' }}>
+        <span aria-hidden style={{ verticalAlign: 'text-bottom' }}>
+          {splitTitle[splitTitle.length - 1]}
+        </span>
         <Tooltip removeFindDomNode content={`${ODH_PRODUCT_NAME} certified and supported`}>
           <img
             style={{ marginLeft: 'var(--pf-global--spacer--xs)' }}
             src="../images/CheckStar.svg"
             alt={`${ODH_PRODUCT_NAME} certified and supported`}
           />
         </Tooltip>
-      )}
+      </span>
+    );
+  }
+
+  return (
+    <CardTitle>
+      <span aria-labelledby={odhApp.spec.displayName} style={{ verticalAlign: 'text-bottom' }}>
```

Comment:
This caused an axe violation -- labelledby needs a role associated -- and reading more into it, it looks like labelledby requires another item to contain that as an id -- so I suspect this is not right. I'm asking PF for more info

Reply:
hmm ok. will just switching it to aria-label work?

Reply:
I believe it also has the same issue -- waiting to hear what PF states.

Reply:
If the reader is smart enough -- it's possible we don't need to do anything.

Reply:
removed aria labels and encased icon in plain button for accessibility